src/components/deck-editor/meta-data-editor/meta-data-editor.tsx

```diff
@@ -0,0 +1,127 @@
+import './meta-data-editor.scss';
+import React from 'react';
+import { Deck } from '../../../models/deck';
+import { Button } from '../../button/button';
+import { TextBox } from '../../text-box/text-box';
+import { BubbleDivider } from '../../bubble-divider/bubble-divider';
+import { DropDown, DropDownOption } from '../../drop-down/drop-down';
+import { Language } from '../../../models/card-content';
+import { TextArea } from '../../text-area/text-area';
+
+// Todo: maybe change Language to be enum instead of type
+enum Languages {
```

Comment:
For future reference, we have this as a type in `card-content.ts`

Reply:
Yeah, that's what I meant, haha.

But if we need to keep it is a type, we could create a Record<Language, string> object in the card-content.ts file and update them together

Reply:
I think at that point an enum is probably better